deployment/terraform/anonymized_database_dump_sheduled_task/docker/dump.sh

```
@@ -0,0 +1,57 @@
+#!/bin/bash -e
+
+export DATE=`date +%Y%m%d-%H%M%S`
+export ANONYMIZED_SNAPSHOT_ID="$DB_INSTANCE_IDENTIFIER-$DATE"
+export TEMPORARY_INSTANCE_ID="$DB_INSTANCE_IDENTIFIER-$DATE"
+
+export SNAPSHOT_ID=$(aws rds describe-db-snapshots --include-shared | jq --arg instance_identifier "$DB_INSTANCE_IDENTIFIER" '[.DBSnapshots[] | select(.DBInstanceIdentifier == $instance_identifier)][-1]' | jq '.DBSnapshotArn' |  tr -d '"\\')
```

Shellcheck

[warning] Declare and assign separately to avoid masking return values.

(SC2155)